.github/workflows/build-base.yml

```diff
@@ -69,6 +69,7 @@ jobs:
             --label "runnumber=${GITHUB_RUN_ID}" \
             --label "stacktag=$(< stack.tag)" \
             --label "eupstag=$(< eups.tag)"
+            < Dockerfile
```

Comment:
Please merge with the commit that removed it; I thought the removal was deliberate!

Is `working-directory: base` still used in this mode?

Reply:
Merged.  Yes working directory base is used.